app/index.js

```diff
@@ -44,6 +44,13 @@ module.exports = class GeneratorLicense extends Generator {
       defaults: (new Date()).getFullYear()
     });
 
+    this.option('licensePrompt', {
```

Comment:
Let's declare as `hide: true` to not bloat the end user help prompt. Documenting this option like you did in the readme should be enough.